libs/flux/build.go

```diff
@@ -450,3 +458,14 @@ func removeEnvVar(env []string, key string) []string {
 	}
 	return env
 }
+
+func setEnvVar(env []string, key string, value string) []string {
```

Comment:
Can you add a case here where it appends the environment variable if it doesn't exist?

Reply:
Done.